### ISSUE TITLE
Localization support

### ISSUE THREAD

**Issue by Anteness** [hyblocker/OpenVR-SpaceCalibrator#7]
Hi, do you mind to add a localization support for multiple languages? I would love to help you translate it! Also I can ask some members from localization project to help.

**Comment by hyblocker** [comment]
Localisation is something I've considered doing alongside a UI update.

**Comment by ColorlessColor** [comment]
Should gettext be introduced as i18n infrastructure?

**Comment by hyblocker** [comment]
> Should gettext be introduced as i18n infrastructure?

No, gettext's license conflicts with this project's license (in a way that would prohibit distribution via Steam).

**Comment by hyblocker** [comment]
implemented in nova. leaving open as the translations themselves aren't done. (if you're fluent in english and another language, and want to help feel free to reach out!)

**Comment by Anteness** [comment]
Sure! How should I assist you?

**Comment by hyblocker** [comment]
join the discord server (https://discord.gg/AV7fsQn9z6) and let me know; localisation work will be handled there for now
